PipelineCommon/Models/BusMessages/RenderingResultMessage.cs

```diff
@@ -18,6 +18,7 @@ public class RenderingResultMessage
 	
 	public int RepoId { get; set; }
 	public List<RenderedFile> RenderedFiles { get; set; } = new List<RenderedFile>();
+	public string FileBase { get; set; }

```

Comment:
BasePath would be best here

Reply:
Or even FileBasePath

Reply:
A slight nitpick otherwise perfect, thanks for putting this together I really appreciate it.

Reply:
@rbnswartz  -> I'm going to change to FileBasePath (4 more chars, but I like the unique name for searchability) and merge to dev.  Assuming green builds on dev, I'll go kick WA-Catalog and see how many messages come through.  We can revert of course if not everything seems to make it then I'd think we're likely ok.